From 80dde745c0f0a382e7e6b71ba24bef35b81e5eec Mon Sep 17 00:00:00 2001
From: Igor Opaniuk <igor.opaniuk@toradex.com>
Date: Wed, 19 Jun 2019 11:47:10 +0300
Subject: [PATCH 6/6] configs: colibri-imx6ull: switch to DM_VIDEO

Use CONFIG_DM_VIDEO=y by default for Colibri iMX6ULL.

Signed-off-by: Igor Opaniuk <igor.opaniuk@toradex.com>
---
 configs/colibri-imx6ull_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/colibri-imx6ull_defconfig b/configs/colibri-imx6ull_defconfig
index 8f09f1b3d6..bb9d6b5707 100644
--- a/configs/colibri-imx6ull_defconfig
+++ b/configs/colibri-imx6ull_defconfig
@@ -77,6 +77,6 @@ CONFIG_USB_GADGET_VENDOR_NUM=0x1b67
 CONFIG_USB_GADGET_PRODUCT_NUM=0x4000
 CONFIG_CI_UDC=y
 CONFIG_USB_GADGET_DOWNLOAD=y
-CONFIG_VIDEO=y
+CONFIG_DM_VIDEO=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_FDT_FIXUP_PARTITIONS=y
-- 
2.13.6

